opencensus/stats/aggregation_data.py

```diff
@@ -52,16 +52,21 @@ def to_point(self, timestamp):
         raise NotImplementedError  # pragma: NO COVER
 
 
```

Comment:
The `BaseAggregationData` class seems a little odd to me as well, each implementation supplies `aggregation_data` upon construction but then doesn't update it, and stores there data in a separate instance variable that they update and return.

Is there a reason for this class to exist?

Reply:
Not that I can tell, besides to show that aggregations should implement `to_point`, but in that case there should be an unimplemented `add_sample` in the class too.

In any case it looks like you can lose the `aggregation_data` attr and property.

Reply:
I just removed `BaseAggregationData` entirely